source/educators/references/grading/learner_view_of_grades.rst

```diff
@@ -1,11 +1,15 @@
 .. _The Learner View of Grades:
 
-********************************************
+##########################################
 The Learner View of Grades (Progress Page)
-********************************************
+##########################################
 
 .. tags:: educator, reference
 
+.. list-table::
```

Comment:
What is the purpose of an empty table here?

I'm not sure what it is supposed to be rendering, and it is created an error in the build.

Reply:
Ok! It was for the ToC which was not included in both docs, but apparently it is not needed, and it's causing the error.

Reply:
@jswope00 tomorrow I'll take a look to the error you're mentioning about reference to an external link. If I am not able to fix it, I'll ask you for help.